index.html

```diff
@@ -22,24 +22,160 @@
     <h1>I'm Majek Glad to see you!</h1>
     <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
     <h3 class="links">LET'S CONNECT</h3>
-  
+  </section>
     <!--Social media accounts-->
+  <footer>
     <ul class="social-icons">
       <li><a href="#"><img src="images/Twitter.jpg" alt="twitter"></a></li>
       <li><a href="#"><img src="images/Linkedin icon.jpg" alt="Linkedin"></a></li>
       <li><a href="#"><img src="images/M.jpg" alt="M"></a></li>
       <li><a href="#"><img src="images/Githhub.jpg" alt="Github"></a></li>
     </ul>
-  </section>
-</body>
-</html>
-
+  </footer>
 
+  <!--works section-->
 
+  <div class="page-2">
+    <div class="container">
+      <a href="#"><img src="images/Snapshoot Portfolio.jpg"></a>
+      <div id="text">
+        <a href="#"><img src="images/Project Title.jpg"></a>  
+      </div>
+        <ul class="column">CANOPY
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/back.jpg"></a></li>
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/year.jpg"></a></li>
+        </ul>
+      <div>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+    </div>
+    <footer>
+      <ul class="work">
+        <li><a href="#">html</a></li>
+        <li><a href="#">css</a></li>
+        <li><a href="#">javaScript</a></li>
+      </ul>
+      <div class="view">
+        <a href="#"><img src="images/Enabled.jpg"></a>
+      </div>
+    </footer>
+  </div>
 
 
+  <div class="page-3">
+    <div class="container-3">
+      <a href="#"><img src="images/Snapshoot Portfolio 2.jpg"></a>
+        <div id="text-3">
+          <a href="#"><img src="images/Project Title 2.jpg"></a>  
```

Comment:
Kindly use an `a` or a `button` tag for the see project button rather than an image. This is more semantic and would grant more flexibility.